src/frontend/src/router/routes.ts

```diff
@@ -24,4 +25,13 @@ export const routes = [
       redirect: get(auth_state) === AuthStates.REGISTERED ? "/posts" : "/login",
     }
   },
+  {
+    name: "/edit-profile",
+    layout: BasicLayout,
+    component: EditProfile,
+    onlyIf: {
+      guard: () => get(auth_state) === AuthStates.NOT_YET_REGISTERED,
```

Comment:
tu ten guard musí byť `get(auth_state) === AuthStates.NOT_YET_REGISTERED`, inak to nefunguje. Editnú profil môže človek až vtedy, keď ho má vytvorený, teda je registrovaný.